src/mcp/prompts/crashlytics/connect.ts

```diff
@@ -80,16 +97,18 @@ Follow these steps to diagnose and fix issues.
   3. Use the 'crashlytics_list_events' tool to get an example crash for this issue.
     3a. Apply the same filtering criteria that you used to find the issue, so that you find an appropriate event.
   4. Read the files that exist in the stack trace of the issue to understand the crash deeply.
-  5. Determine the root cause of the crash.
-  6. Write out a plan using the following criteria:
-    6a. Write out a description of the issue and including
+  5. Determine possible root causes for the crash.
+  6. Critique your own determination, analyzing how plausible this scenario is.
+  7. Choose the most likely root cause given your analysis.
+  8. Write out a plan for the most likely root cause using the following criteria:
+    8a. Write out a description of the issue and including
         * A brief description of the cause of the issue
-        * A determination of your level of confidence in the cause of the issue
+        * A determination of your level of confidence in the cause of the issue using your analysis.
         * A determination of which library is at fault, this codebase or a dependent library
         * A determination for how complex the fix will be
-    6b. The plan should include relevant files to change
-    6c. The plan should include a test plan to verify the fix
-    6d. Use the following format for the plan:
+    8b. The plan should include relevant files to change
+    8c. The plan should include a test plan for how the user might verify the fix
+    8d. Use the following format for the plan:
```

Comment:
Let's also set an upper cap of how many causes and test cases the agent should consider. The more we constraint, model could be pick the most important ones relevant.

Reply:
I don't see the change for this. Could I be misreading something?